### PR TITLE
Add deledges storm command

### PR DESCRIPTION
**Pull request by Cisphyx**

### CODE REVIEW

File: synapse/lib/storm.py [hunk 1]
```diff
@@ -2926,3 +2926,62 @@ async def execStormCmd(self, runt, genr):
...
+        # Delete all "foo" light edges from an inet:ipv4
+        inet:ipv4=1.2.3.4 | deledges foo
+
+        # Delete light edges with any verb from a node
+        inet:ipv4=1.2.3.4 | deledges *
+
+        # Delete all "foo" light edges to an inet:ipv4
+        inet:ipv4=1.2.3.4 | deledges foo --n2
+    '''
+    name = 'deledges'
```
Comment (invisig0th) [inline synapse/lib/storm.py:2945]: Lets do a quick goback about potentially namspacing this command name

**Review by invisig0th — CHANGES_REQUESTED** [review]

File: synapse/lib/storm.py (hunk 1 above)
Comment (Cisphyx) [reply synapse/lib/storm.py]: Updated to `edge.del`

File: synapse/lib/storm.py [hunk 2]
```diff
@@ -2926,3 +2926,62 @@ async def execStormCmd(self, runt, genr):
...
+        # Delete all "foo" light edges from an inet:ipv4
+        inet:ipv4=1.2.3.4 | edge.del foo
+
+        # Delete light edges with any verb from a node
+        inet:ipv4=1.2.3.4 | edge.del *
+
+        # Delete all "foo" light edges to an inet:ipv4
+        inet:ipv4=1.2.3.4 | edge.del foo --n2
+    '''
+    name = 'edge.del'
```
Comment (invisig0th) [inline synapse/lib/storm.py:2945]: lets pluralize to `edges.del` ( sorry about the nit picking, i promise to mash approve :D )

**Review by invisig0th — CHANGES_REQUESTED** [review]

File: synapse/lib/storm.py (hunk 2 above)
Comment (Cisphyx) [reply synapse/lib/storm.py]: haha, it's all good, updated to `edges.del`

**Review by invisig0th — APPROVED** [review]

**Pull request merged**